dialog/fileitem.go

```diff
@@ -39,7 +45,8 @@ func (i *fileDialogItem) CreateRenderer() fyne.WidgetRenderer {
 	}
 }
 
-func (i *fileDialogItem) setLocation(l fyne.URI, dir, up bool) {
+func (i *fileDialogItem) setLocation(l fyne.URI, dir, up bool, id int) {
```

Comment:
Should this function be renamed since it's doing more than setting location? Like maybe just `update` or `updateForURI`?

Reply:
Yes. I can use the same way as choose function.